### PR TITLE
Added related projects

### PR DESCRIPTION
**Pull request by ndevillard**
Added related projects

### CODE REVIEW

**Review by athoelke — CHANGES_REQUESTED** [review]

File: related-projects.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  [https://www.trustedfirmware.org/projects/trusted-services/](https://www.trustedfirmware.org/projects/trusted-services/)
+
+- PARSEC is a Rust-based Linux microservice based on the Crypto API
+  [https://parsec.community](https://parsec.community)
+
+- Oberon PSA Crypto offers a way to support both hardware and optimized software crypto routines under the PSA Crypto API
+  [https://www.oberon-psa-crypto.ch/](https://www.oberon-psa-crypto.ch/)
+
+- RIOT OS uses the Crypto API at a system-level for all crypto needs
+  [https://www.riot-os.org/](https://www.riot-os.org/)
```
[the remark this answers is not in the document]
Comment (ndevillard) [reply related-projects.md:28]: ok. Rephrased and added link to paper

File: related-projects.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+## Hardware driver implementations
+
+- The Silicon Labs SDK uses PSA driver API conventions
```
[the remark this answers is not in the document]
Comment (ndevillard) [reply related-projects.md:33]: Ok to add text. Eventually, the driver interface really belongs in this repo as it is not Mbed TLS specific.

File: related-projects.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  [https://www.silabs.com/developers/gecko-software-development-kit](https://www.silabs.com/developers/gecko-software-development-kit)
+
+- The Nordic SDK uses PSA driver API conventions
+  [https://github.com/nrfconnect/sdk-nrf/tree/main/samples/crypto/psa_tls](https://github.com/nrfconnect/sdk-nrf/tree/main/samples/crypto/psa_tls)
+ 
+ 
+## API usage
+
+- Mbed TLS implements TLS stack for embedded usage, based on Crypto API
+  [https://mbed-tls.readthedocs.io/en/latest/](https://mbed-tls.readthedocs.io/en/latest/)
```
[the remark this answers is not in the document]
Comment (ndevillard) [reply related-projects.md:43]: added

File: related-projects.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (athoelke) [reply related-projects.md:35]: Agreed, and we might want an issue in psa-api to document that intention.

File: related-projects.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+-->
+
+# Known implementations and projects related to the PSA Certified APIs
+
+Updated Dec 2022.
+ 
+## Software implementations
+
+- The reference implementation for PSA Crypto API can be found inside Mbed TLS:
+  [https://mbed-tls.readthedocs.io/en/latest/](https://mbed-tls.readthedocs.io/en/latest/)
```
Comment (athoelke) [inline related-projects.md:13]: On other pages on this website, the URL is shown without the "https://" prefix, and without any trailing "/":

```suggestion
  [mbed-tls.readthedocs.io/en/latest](https://mbed-tls.readthedocs.io/en/latest/)
```

This reduces the visual clutter and makes it easier to see the important part of the URL.

I think this page will look better if this is done here too.

File: related-projects.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+- Other PSA Certified APIs are included in the TF-M project:
+  [https://www.trustedfirmware.org/projects/tf-m/](https://www.trustedfirmware.org/projects/tf-m/)
+
+- Trusted Services implements those services in a Trusted Execution Environment on Cortex A processors
```
Comment (athoelke) [inline related-projects.md:18]: There is no trailing colon here.

Can we make all of the list items consistent: either using a trailing colon before the link, or ending with "." and introduce the link with "See"

**Review by athoelke — CHANGES_REQUESTED** [review]
I spotted a few more items, after previewing the page, to improve consistency.

File: related-projects.md (hunk 5 above)
Comment (ndevillard) [reply related-projects.md]: done

File: related-projects.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+ 
+## API usage
+
+- Mbed TLS implements TLS stack for embedded usage, based on Crypto API. Mbed TLS also uses the Secure Storage API for the Crypto implementation of key storage.
+  [https://mbed-tls.readthedocs.io/en/latest/](https://mbed-tls.readthedocs.io/en/latest/)
+
+- WolfSSL can consume PSA-compliant crypto backends
+  [https://www.wolfssl.com/platform-security-architecture-psa-crypto-api-support-wolfssl/](https://www.wolfssl.com/platform-security-architecture-psa-crypto-api-support-wolfssl/)
+
+- [t_cose](https://github.com/laurencelundblade/t_cose) uses PSA Crypto APIs for portability.
```
[the remark this answers is not in the document]
Comment (ndevillard) [reply related-projects.md]: done

**Pull request merged**